src/components/common/NavBar.tsx

```diff
@@ -19,90 +19,69 @@ interface LiStyle {
 
 function NavBar() {
   const router = useRouter();
-  const dayPeriod = useRecoilValue(dayInfo);
-  const weekPeriod = useRecoilValue(weekInfo);
+  const dayPeriod = getTodayDate(0);
   const [pickedMenu, setPickedMenu] = useState<isMenuType>('Today');
-  const weekDomain = `${weekPeriod[0]}-${weekPeriod[6]}`;
-  const [myPageIsOpened, setMyPageIsOpened] = useState(false);
-  const [hamburgerMenu, setHamburgerMenu] = useRecoilState(RoutineBoxIsOpened);
+  const [hamburgerMenu, setHamburgerMenu] = useState(false);
 
   useEffect(() => {
     setHamburgerMenu(false);
-    setMyPageIsOpened(false);
   }, []);
 
   const handleHamburgerMenu = () => {
-    setHamburgerMenu(false);
-
-    if (hamburgerMenu === true) {
-      setMyPageIsOpened(false);
-    } else {
-      setHamburgerMenu(true);
-    }
-  };
-  const handleModalToggle = () => {
-    if (hamburgerMenu) {
-      setMyPageIsOpened(false);
-    } else if (!hamburgerMenu) {
-      myPageIsOpened ? setMyPageIsOpened(false) : setMyPageIsOpened(true);
+    hamburgerMenu === false ? setHamburgerMenu(true) : setHamburgerMenu(false);
+    if (router.pathname === '/') {
+      setHamburgerMenu(false);
```

Comment:
크 훨씬 깔끔해진 것 같아요! 이대로도 좋지만 몇 가지 첨언하자면,
1. router.pathname이 '/'인 경우에는 위의 연산이 불필요할 것 같아서 그 경우를 먼저 처리해도 좋을 것 같아요!
2. hamburgetMenu의 값에 따라 boolean값을 반대로 세팅하고 싶은 거라면(false일때 true, true일 때 false) prev를 사용하는 편이 좀 더 의미가 명확할 것 같아요! 
```suggestion
    if (router.pathname === '/') {
      setHamburgerMenu(false);
      return;
    }
    setHamburgerMenu(prev => !prev);
```